onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/FareProduct.java

```diff
@@ -124,8 +124,14 @@ public void setCurrency(String currency) {
   @Override
   public AgencyAndId getId() {
     String riderCategoryId = Optional.ofNullable(riderCategory).map(c -> c.getId().getId()).orElse(null);
-    String fareContainerId = Optional.ofNullable(fareContainer).map(c -> c.getId().getId()).orElse(null);
-    return FareProductFieldMappingFactory.fareProductId(fareProductId.getAgencyId(), fareProductId.getId(), riderCategoryId, fareContainerId);
+    String fareMediumId = Optional.ofNullable(fareMedium).map(c -> c.getId().getId()).orElse(null);
+    return FareProductFieldMappingFactory.fareProductId(
+            fareProductId.getAgencyId(),
+            fareProductId.getId(),
+            getName(),
```

Comment:
Why is the name now part of the ID? Accoring to the spec this isn't part of the primary key.

Reply:
https://docs.google.com/document/d/19j-f-wZ5C_kYXmkLBye1g42U-kvfSVgYLkkG5oyBauY/edit?pli=1

Good catch, I was basing it off of this doc, but the section that includes name isn't grayed, and there's an unaddressed comment that brings up why including the name is probably a bad idea. Maybe that was what caused them to remove that in the final spec? I'll get rid of it.